sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -55,15 +55,21 @@ object CosmosAccountConfig {
     parseFromStringFunction = accountEndpointUri => accountEndpointUri,
     helpMessage = "Cosmos DB Account Key")
 
+  val CosmosConsistency = CosmosConfigEntry[String](key = "spark.cosmos.accountConsistency",
+    mandatory = false,
+    parseFromStringFunction = consistency => consistency,
```

Comment:
please do the validation String -> ConsistencyLevel here.

Reply:
No longer relevant, doesn't exist anymore